frontend/src/Components/CareersForm/CareersForm.jsx

```diff
@@ -6,26 +6,31 @@ import { string as yupstring, object as yupobject } from "yup";
 import { useForm } from "react-hook-form";
 import { sendEmail, sendFile } from "Utils/Requests";
 import { DropzoneArea } from "material-ui-dropzone";
+import { CustomSnackbar } from "Components";
 
 import "./CareersForm.scss";
 
-const useInputStyles = makeStyles(theme => ({
+const MAX_FILE_SIZE = 3000000;
+
+//underlining
+const useInputStyles = makeStyles((theme) => ({
   root: {
     "& label.Mui-focused": {
-      color: "black"
+      color: "black",
     },
     "& .MuiFilledInput-underline:after": {
-      borderBottomColor: "black"
-    }
-  }
+      borderBottomColor: "black",
+    },
+  },
```

Comment:
You changed these but just out of curiosity, It's correct to have comma's with no data following?

Reply:
I'm not entirely sure, I believe prettier did this